client/app/components/preprint-form-header/template.hbs

```diff
@@ -3,7 +3,12 @@
 <!-- template-lint block-indentation=false -->
 
 <div class="section-header">
-    {{name}}
+    {{#if isOpen}}
+        {{displayName}}
+    {{else}}
+        <a href="#">{{displayName}}</a>
```

Comment:
Ok! Also I realized the header just looks like a link on the preprints form but it is not actually a link.